addons/web/static/tests/core/utils/numbers_tests.js

```diff
@@ -0,0 +1,64 @@
+/** @odoo-module **/
+
+import { roundPrecision, roundDecimals } from "@web/core/utils/numbers";
+
+QUnit.module("utils", () => {
```

Comment:
Can't drop the legacy versions of `roundPrecision` and `roundDecimals` because the legacy `float_is_zero` is being used in the `PropertiesMixin`.